source/_integrations/plugwise.markdown

```diff
@@ -3,44 +3,50 @@ title: Plugwise
 description: Plugwise Smile platform integration.
 ha_category:
   - Climate
+  - Binary Sensor
   - Sensor
   - Switch
-  - Water Heater
 ha_iot_class: Local Polling
 ha_release: 0.98
 ha_codeowners:
   - '@CoMPaTech'
   - '@bouwew'
+  - '@brefra'
 ha_config_flow: true
 ha_domain: plugwise
 ---
 
-This enables [Plugwise](https://www.plugwise.com) components with a central Smile gateway to be integrated. This integration talks locally to your **Smile** interface, and you will need its password and IP address.
-The platform supports [Anna](https://www.plugwise.com/en_US/products/anna), [Adam (zonecontrol)](https://www.plugwise.com/en_US/zonecontrol), [P1](https://www.plugwise.com/en_US/products/smile-p1) Smile products and the [Stretch](https://www.plugwise.com/nl_NL/het-systeem) product. See below list for more details.
+This enables [Plugwise](https://www.plugwise.com) components with a central Smile gateway or USB-stick and components to be integrated. For USB you will need to know which port your stick is connected to. For the networked Plugwise Smile you will need its password and IP address.
+
+This integration supports [Anna](https://www.plugwise.com/en_US/products/anna), [Adam (zonecontrol)](https://www.plugwise.com/en_US/zonecontrol), [P1](https://www.plugwise.com/en_US/products/smile-p1) Smile products and other Plugwise products through the [Stretch](https://www.plugwise.com/nl_NL/het-systeem) product or using the USB-stick. See below list for more details.
```

Comment:
This integration supports [Anna](https://www.plugwise.com/en_US/products/anna), [Adam (zonecontrol)](https://www.plugwise.com/en_US/zonecontrol), [P1](https://www.plugwise.com/en_US/products/smile-p1) Smile products and various legacy devices (Circle, Stealth, etc.) via the [Stretch](https://www.plugwise.com/nl_NL/het-systeem) or the Stick. See below list for more details.